cli/root.go

```diff
@@ -147,6 +151,8 @@ func init() {
 	RootCmd.PersistentFlags().StringVarP(&manager.ServiceManager, "svcmgr", "m", "", fmt.Sprintf("service manager, must be one of: %s", strings.Join(backends, ", ")))
 	RootCmd.PersistentFlags().DurationVarP(&manager.Before, "before", "t", mgr.DefaultBefore, "how long before certificates expire to start renewing (in duration format)")
 	RootCmd.PersistentFlags().DurationVarP(&manager.Interval, "interval", "i", mgr.DefaultInterval, "how long to sleep before checking for renewal (in duration format)")
+	RootCmd.PersistentFlags().DurationVarP(&manager.IntervalSplay, "interval.splay", "", 0*time.Second, "a rng value of [0..interval.splay] to add to each interval to randomize wake time")
+	RootCmd.PersistentFlags().DurationVarP(&manager.InitialSplay, "initial.splay", "", 0*time.Second, "a rng value of [0..initial.splay] to add to each the first wake period to randomize instances")
```

Comment:
`to add to each the first wake period` should be `to add to the first wake period` it looks like.
```suggestion
	RootCmd.PersistentFlags().DurationVarP(&manager.InitialSplay, "initial.splay", "", 0*time.Second, "a rng value of [0..initial.splay] to add to the first wake period to randomize instances")
```

Reply:
ack, making the change shortly.